### DISCUSSION TITLE
Stripe API Keys upgrade question

### DISCUSSION THREAD

**Discussion by jxs714** [pretix/pretix#4502] [Q&A] [answered]
Hey guys,

I am using Stripe with Pretix and realize that I have an older version of API Keys in Stripe. Didn't realize this was a thing. Is it safe to update this at anytime? Will this break anything? Screenshot attached

![Stripe API upgrade Screenshot](https://github.com/user-attachments/assets/08550064-2e6c-4e81-b3c2-1fe6e7da0f07)

**Comment by raphaelm** [comment, marked as answer]
I think it will work fine, but I can't say for sure, I thin pretix currently makes requests with API version 2023-10-16 so every change until than should be safe.

**Reply by jxs714** [reply]
Thanks for this! Are there additional features that will appear during checkout when API's are upgraded such as 3D Secure authentication with bank, etc.

**Reply by raphaelm** [reply]
No, usually the API version only affects the API, not visible things
